views/actions/session.js

```diff
@@ -48,7 +63,15 @@ function createSession(request) {
           dispatch(push('/packages'));
         });
       } else {
-        dispatch(authError());
+        if (response.status === 422 &&
+          response.statusText === 'Unprocessable Entity') {
+          dispatch(addRegistrationErrors([{
+            message: REGISTRATION_EMAIL_ERROR,
+            key: errorKey++,
+          }]));
```

Comment:
I don't think you can assume what error the server will return. It is better to display what the server returns.